Source/CMakeSettings.json

```diff
@@ -0,0 +1,15 @@
+﻿{
```

Comment:
My question is whether keeping this one version in the project is a sensible thing to do. Normally I'd prefer these to stay local to the dev's machine.

Reply:
I'm fine removing it and adding to gitignore.
The reason I added it is the default is /out rather than matching msbuild /bin.

Reply:
My preference would be to add the entry to the ignore file, but obviously I don't have a strong opinion and it doesn't really affect me. If you think others will find it useful, go ahead.

Reply:
I'll remove it for now since I'm not super familiar with CMake on Windows.
It didn't really *add/make anything easier* by changing the /out to /bin.
Ultimately it doesn't matter, so removing it.